src/components/Layouts/index.js

```diff
@@ -202,6 +226,16 @@ class Layout extends React.Component {
 
               <HeaderGlobalBar>
                 {/* {isInternal ? null : <GlobalSearch />} */}
+                {this.state.isSearchOpen ? <GlobalSearch /> : 
```

Comment:
Instead of removing the entire element from the screen, can we toggle a specific class? Something like `bx--header__action--search--active` 

That way we can set the width to 0 and make its visibility hidden when closed and animate the width to 448px on click.